### PR TITLE
Feature/plop template

### PR DESCRIPTION
**Pull request by braczkow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,31 @@ The Android application devstarter project template that can be used to start wr
 
 6. Enjoy!
 
+## Templates
+
+As the MVI architecture used for presentation layer is quite complex, a helper templates solution has been provided.
+Its main goal is to auto-generate all the boilerplate code that constitutes a fully-functional screen.
+
+Templates solution was developerd with help of node.js [plop](https://plopjs.com/) module: .
```
Comment (MDikkii) [inline README.md:58]: typo `developerd`

File: .plop-templates/mvi/{{pascalCase name}}ViewState.kt.hbs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package {{ basePackage }}.{{camelCase name }}
+
+import java.io.Serializable
+
+class {{pascalCase name}}ViewState() : Serializable {
+    sealed class PartialState {
+    }
+}
```
Comment (MDikkii) [inline .plop-templates/mvi/{{pascalCase name}}ViewState.kt.hbs:8]: Missing new line

**Review by MDikkii — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (braczkow) [reply README.md]: Fixed

File: .plop-templates/mvi/{{pascalCase name}}ViewState.kt.hbs (hunk 2 above)
Comment (braczkow) [reply .plop-templates/mvi/{{pascalCase name}}ViewState.kt.hbs]: Fixed

**Review by MDikkii — APPROVED** [review]

**Pull request merged**